gene_finder.py

```diff
@@ -45,11 +51,25 @@ def get_reverse_complement(dna):
     >>> get_reverse_complement("CCGCGTTCA")
     'TGAACGCGG'
     """
-    # TODO: implement this
-    pass
-
-
-def rest_of_ORF(dna):
+    # Step 1 get complement
+    counter_1=0
+    complement_dna = ''
+    while counter_1 < len(dna):
+        complement_dna = complement_dna + get_complement(dna[counter_1])
+        counter_1 = counter_1 + 1
+    # Step 2 reverse it
+    reverse_complement = ''
+    counter_2=len(dna)-1
+    while counter_2 >= 0:
+        reverse_complement = reverse_complement + complement_dna[counter_2]
+        counter_2 = counter_2 - 1
+    return reverse_complement
```

Comment:
Going Beyond : Try using list comprehensions.

`return [get_complement(c) for c in dna[::-1]]`

for simple functinos like this, this one line of code is more readable than 10 lines of code that does the same thing